translations/monero-core_de.ts

```diff
@@ -954,22 +954,22 @@
     <message>
         <location filename="../pages/Settings.qml" line="102"/>
         <source>Create view only wallet</source>
-        <translation type="unfinished"></translation>
+        <translation>Schreibgeschütze Wallet erstellen</translation>
```

Comment:
I get your point, read-only is not writing-forbidden, true. "Beobachten" sounds too much like spying, "beschränkt" alone doesn't make clear, what the limitation is. Often you see the terms "Lesezugriff", "Leseberechtigung" or "nur lesen" in such cases, I guess? Maybe something like "Wallet mit beschränktem Lesezugriff erstellen" or "Wallet nur mit Leseberechtigung erstellen" is clearer?